### DISCUSSION TITLE
Help configuring plugin

### DISCUSSION THREAD

**Discussion by Nodens30** [colinbendell/homebridge-blink-for-home#92] [Q&A]
I've just bought a Hoobs box and want to get my Blink cameras working with Apple Homekit

I installed the plugin and click the config button but don't see the form to enter the account details. Instead I get code as shown in the attached. 

I've entered my account details and saved. There is no trigger of the 2FA for the pin. 

I've tried uninstalling and re-installing but am now stuck and need advice please? 
![Screenshot 2022-01-27 at 12 27 31](https://user-images.githubusercontent.com/98529500/151359285-f7ab78c2-61fb-428d-939e-6146c5d3c718.png)

**Comment by cpena28** [comment]
Try this:

{
    "accessories": [],
    "platforms": [
        {
            "username": "Email",
            "password": "Password",
            "pin": "2FA code",
            "platform": "Blink"
        }
    ]
}
